axelrod/tests/unit/test_ann.py

```diff
@@ -29,15 +39,15 @@ def test_strategy(self):
 class TestEvolvedANNvsCooperator(TestHeadsUp):
     def test_rounds(self):
         self.versus_test(axelrod.EvolvedANN(), axelrod.Cooperator(),
-                         [C, D, D, C, D], [C] * 5)
+                         [C, C, C, C, C], [C] * 5)
 
 
 class TestEvolvedANNvsDefector(TestHeadsUp):
     def test_rounds(self):
         self.versus_test(axelrod.EvolvedANN(), axelrod.Defector(),
-                         [C, D, D, D, D], [D] * 5)
+                         [C, C, D, D, D], [D] * 5)
 
 class TestEvolvedANNvsTFT(TestHeadsUp):
     def test_rounds(self):
         self.versus_test(axelrod.EvolvedANN(), axelrod.TitForTat(),
-                         [C, D, D, C, C], [C, C, D, D, C] * 5)
+                         [C] * 5, [C] * 5)
```

Comment:
Could we have some specific tests for the other two trained ANN strategies please.

Also: can you include a test for the exception that gets raised when the weights in the dataset have incorrect size.